component/tasks.js

```diff
@@ -89,6 +89,10 @@ class Tasks {
     const workspace = this;
     fsUtility.readDataSource(workspace, cb);
   }
+  refreshMiddleware(cb) {
+    const workspace = this;
```

Comment:
tasks.js is a mixin class for workspace.js. I just want to be clear here so that other developers will understand that.